README.md

```diff
@@ -62,6 +62,24 @@ from internal types to those the API expects to return.
 This approach to pluggable interfaces is similar to other core Kubernetes APIs
 like +NetworkPolicy+, +Ingress+ and +CustomMetrics+.
 
+## Ecosystem
+
+### Implementations
+
+* **Flagger:** progressive delivery operator (flagger.app)
+* **Rio:** application deployment engine (rio.io)
+* **Service Mesh Hub:** unified dashboard (solo.io/products/service-mesh-hub)
+* **SuperGloo:** mesh orchestration (supergloo.solo.io)
+
+### Integrations
+
+* **Maesh:** simpler service mesh (mae.sh)
+* **Linkerd:** ultralight service mesh (linkerd.io)
+* **Istio*:** connect, secure, control, observe (deislabs/smi-adapter-istio)
```

Comment:
Unsure if we want to link to the adaptor; the full link may be more helpful for finding it?  https://github.com/deislabs/smi-adapter-istio

Reply:
The link should indeed be better but would certainly link it here imo, otherwise folks will start looking through the Istio docs